actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -88,7 +88,12 @@ def translate(key, options = {})
           raise e if raise_error
 
           keys = I18n.normalize_keys(e.locale, e.key, e.options[:scope])
-          content_tag('span', keys.last.to_s.titleize, :class => 'translation_missing', :title => "translation missing: #{keys.join('.')}")
+          title = "translation missing: #{keys.join('.')}"
+          interpolation_options = options.except(:default)
+          if interpolation_options.present?
```

Comment:
`present?` is generally overused. Let's just go with good old `empty?`.

Aren't the `interpolation_options` just `interpolations`? I'd love a rename if so :grin: